src/components/coursesSection/course/Course.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import courseStyle from "./Course.module.css";
+
+function Course(props) {
```

Comment:
it is better to receive the props in this format "function Course({ courseData })"

Reply:
Done